### ISSUE TITLE
Donorbox implementation

### ISSUE THREAD

**Issue by jasloe** [aarome/aarome#359]
Documentation here: https://donorbox.zendesk.com/hc/en-us/articles/360020293172-Is-there-a-demo-site-that-I-can-use-to-test-out-a-Donorbox-organization-account

**Comment by jasloe** [comment]
After attempting to attach inline js the "old way", I learned that Drupal 8/9 has deprecated this approach. 

See: https://www.drupal.org/project/drupal/issues/2391025

And: https://www.drupal.org/node/2274843#inline

Drupal recommends using ```hook_page_attachments```, e.g.: 

```
function donorbox_page_attachments(array &$attachments) {
  $attachments['#attached']['html_head'][] = [
    [
      '#type' => 'html_tag',
      '#tag' => 'script',
      '#attributes' => ['src' => 'https://dboxm.com/install-popup-button.js'],
    ],
    'donorbox',
  ];
}
```

Cheaper options include: 

- Try this: https://www.drupal.org/project/attachinline
- Place the markup in a Twig template

**Comment by cchristopherhowardd** [comment]
I’ll be happy to see the email design soon. I’ll also bust out some pencil and paper to sketch a simple layout. My understanding is that AAR’s development team and the Point Studio people expect you and me to come up with a design. I imagine that they would like to see a page using the pop-up option for Donorbox and a side option.

The gala stream is coming from Vimeo (not Livestream, which people are using as a verb, not a platform). You could probably grab <iframe> code from https://vimeo.com/aarome or from any Vimeo video, one would think.

This project is obviously last-minute, and while it would be useful to create a page that we can use again, the immediate concern is getting a functioning page (which would live at aarome.org/benefit) by October 27.

**Comment by cchristopherhowardd** [comment]
From Point Studio:

"We can provide a sample Vimeo live feed for you to test while building the page design. When would you like to try this out? I can generate the embed code, but will need to coordinate when a feed should go live."

**Comment by jasloe** [comment]
How about this? 

- Receive Wendy's feedback
- style the layout per above
- configure campaign and forms on the Donorbox portal
- test livestream embed and form together on production

Do you already have an organization account for Donorbox set up for the Academy?

**Comment by cchristopherhowardd** [comment]
That sounds like a good schedule. I sent Wendy JPEGs of my world-famous website sketches a couple of hours ago. No response yet. That's fine because I want to wrap up the October email soon.

AAR development is deciding on whether or not to use Donorbox this week, but they seems to leaning toward yes. I imagine that there's no official account _per se_ until some kind of contract is signed.

**Comment by jasloe** [comment]
Sounds good. By the way, these are not sequential steps. I can get the basics bootstrapped and posted to production by tomorrow so there's something to look at.

**Comment by cchristopherhowardd** [comment]
Should I share my drawings? Wendy hasn't responded, surprisingly.

I like the first one the best. It accounts for a pop-up donation.

![Alternate draft pop-up](https://user-images.githubusercontent.com/38765846/94622121-94791f00-027f-11eb-8088-946ef4647a1c.jpg)

The other two anticipate a sidebar donation form that's minimized upon first page load but expands when clicked.

![Draft side donate button](https://user-images.githubusercontent.com/38765846/94622203-c2f6fa00-027f-11eb-9778-bc083025bc2a.jpg)

![Draft side donate form](https://user-images.githubusercontent.com/38765846/94622218-c8544480-027f-11eb-912a-7f0217d13c4e.jpg)

It could be that a combination of both might work, too. What do you think?

**Comment by jasloe** [comment]
First of all, what is that background? I love it. The first one works in the abstract but I think a smaller video viewport is better. Live videos artifact a lot at wider viewports. Also, the call to action (I hate that term) buttons are buried. I like option 2. 

In general, I want to steer clear of the popup. There's not a lot we can do with positioning. 

But let me try building out option 2 for starters and if people want to see another option, I can do that, too.

Nice work!

**Comment by cchristopherhowardd** [comment]
The background is a Marimekko duvet cover, an anniversary gift to my wife two years ago. Two years ago the day actually--it's our fourth anniversary today.

The pop-up as demonstrated on the call opened in the middle of the screen and darkened the screen by 50 percent (or whatever). What's your concern with positioning?

I look forward to seeing option 2. It does allow for the screen not to be darkened as people are donating--that's a plus.

**Comment by cchristopherhowardd** [comment]
Wendy from development wrote back this afternoon. I've pasted relevant information below. Her ideas assume a pop-up instead of a sidebar. 

I am attaching some ideas I had too - the main differences are: Instead of a VB home page button, can we make a PROGRAM button and link it to a PDF of the digital program for the evening? Also, I was thinking it would be good to have the headlining names that will be featured in the program included on the page somewhere - theaster gates, mary beard, jason moran, jhumpa lahiri, anthony doerr, basil twist (See my sketch attached). I thought it could take the form of a graphic that Open would create (similar to the evite). It could appear to the right or the left of the screen. Just the honorees and the names of the major "performers" for the evening.

![IMG_5114](https://user-images.githubusercontent.com/38765846/94846733-9d88fe00-03ef-11eb-94c0-203411582228.jpg)

**Comment by cchristopherhowardd** [comment]
The PDF program that she refers to is below. It's the most recent version. The branded image that Wendy proposes would probably look similar.

[ad73_FallGala_092920ck.pdf](https://github.com/aarome/aarome/files/5314032/ad73_FallGala_092920ck.pdf)

**Comment by cchristopherhowardd** [comment]
Other instructions and preferences:

1. Reiterate that the URL should be aarome.org/benefit but the headline should be 2020 AAR VIRTUAL BENEFIT

2. Video on left side of screen

3. Text on first button can read AUCTION but we need to add the credit somewhere on the webpage: Auction powered by [Artsy logo]

4. Have a button that reads DONATE that will click open/pop up to the side of the video screen to show the simple donation form

5. Text on third button can read PROGRAM and link to the program PDF

6. What was previously a button for VIRTUAL BENEFIT HOME would probably work as a sentence with a few words that link to aarome.org/support/virtual-benefit. In fact, this sentence (something like "Visit the **2020 Virtual Benefit homepage** to learn more about this year's event.") and the "Auction powered by [Artsy logo]" line could both be in a text box underneath the buttons. I feel like we may have additional information to put in this text box as we get closer to the Oct 27 event.

**Comment by cchristopherhowardd** [comment]
You should have received an email from Donorbox, which will allow you to create an account and look under the hood. The embed code stuff is found at https://donorbox.org/org_admin/forms/199099?embed=embed_form. Wendy has asked AAR finance if we have Stripe and PayPal accounts. 

Is there any chance you might have a draft page to review today?

**Comment by jasloe** [comment]
Thanks for setting that up. Pulling something (rough) together now. Should have it posted by 4.

**Comment by jasloe** [comment]
Hm...having a little trouble getting the Donorbox.js asset to attach. Had it working on Thursday...not sure what's happening.

Quick question: for the sidebar, did you want to directly drop in the image from Open or have styled text there? I can't remember what we discussed on Friday.

**Comment by cchristopherhowardd** [comment]
Open hasn't yet designed anything for this spot. You could use a page from the PDF invite thing as a placeholder. I don't know exactly what kind of information they want here, and they may not know themselves until they see the draft page.

**Comment by cchristopherhowardd** [comment]
Could this GIF be the sidebar?

![AD73_FallGala_FirstImage](https://user-images.githubusercontent.com/38765846/95141503-64c68d00-073f-11eb-82e3-0caac510fc7b.gif)

**Comment by jasloe** [comment]
I'm working on a proposal that I think will work. First step is getting this damn popup working! Just got into office and will report back in an hour or so.

**Comment by jasloe** [comment]
Ok got it working.

**Comment by cchristopherhowardd** [comment]
Great. Wendy is very anxious about testing the live video stream ASAP. Like, today. She wants to send an email blast invitation for the gala that states the URL of the event will be aarome.org/benefit. Before we can do that, she needs to know for absolute certain that we can stream the live (but mostly prerecorded) Vimeo video program on the AAR website. Can this test be done today?

Once this test is done and the email blast sent, we'll need a link forwarder so that anyone who types in aarome.org/benefit will be redirected to https://www.aarome.org/support/virtual-benefit (until the former page is ready).

**Comment by jasloe** [comment]
Donate popup works now. I think there's a way to set up a dummy processor, no? How do we coordinate a livestream test?

I quickly roughed out another idea this morning at https://staging.aarome.org/benefit

I suggest compiling the call-to-action buttons, program information, and live stream into a one-off, branded, single page presentation. This could be used as both an event promotion asset and possibly contain all or some of the information at https://www.aarome.org/support/virtual-benefit.

There are no media queries at the moment so will fall apart on mobile. But easy to add.

Donate button can drive user to donate page; program button to either pdf or drop user down into styled text at bottom. Lots of options.

Background gradients can be animated using style sheet transitions.

**Comment by cchristopherhowardd** [comment]
I'll take a look after lunch. 

To test the stream with Point Studio, can you contact Christina Choriatis and Josette Persson directly? Their emails are christina@pointstudio.tv and josette@pointstudio.tv, respectively.

**Comment by jasloe** [comment]
Will do. Note what I posted to staging is just an rough, incomplete idea. We'd have to hash out a few details.

**Comment by cchristopherhowardd** [comment]
OK. Just a minor note that Point Studio is using Vimeo, not Livestream, for the gala. When people use the word "livestream" they're not referring to that specific platform but to the general notion of video of a live event being streamed over the internet. So we don't need Livestream logos or branding.

**Comment by jasloe** [comment]
Yeah, I just put the Livestream in there because I wanted something AAR branded.

Dropped the embed code provided by Point into the template. Seems to work fine. (There's really not much to test per se.) 

Let me know which direction you want to go. Up for making something special or just going with the regular website template.

**Comment by cchristopherhowardd** [comment]
That's a Youtube stream at https://staging.aarome.org/benefit, no? Didn't Point Studio give a Vimeo link?

**Comment by cchristopherhowardd** [comment]
WEIRD

**Comment by cchristopherhowardd** [comment]
Wendy anticipates 500 to 1,000 viewers.

**Comment by jasloe** [comment]
That's good to know. How long is the event? 

To ensure reliable performance, we'll have to move the web server from the Apache web server to Nginx, as the latter handles concurrency much better. And also tune the caching to cut down on database requests which can tax the server's memory and CPU.

**Comment by jasloe** [comment]
Also, restating that the live stream embed as provided by Point seems to work fine. We still need to test the entire Donorbox payment process with a live or dummy payment gateway.

Considering an alternative so not to burn a bunch of time messing with server configuration:

- Agree on new design direction (in the next day or two) using Open flyer as basis for a single page "brochure" site for the benefit, including live stream, donation popup, program information
- Hand code above and post as static HTML file served from an S3 bucket at https://www.aarome.org/benefit
- Advantage of above is there are no moving parts: just vanilla html/css/javascript. No database connections, no memory or CPU issues.

**Comment by jasloe** [comment]
Urgh....Donorbox does not provide a dummy processor. 

https://donorbox.zendesk.com/hc/en-us/articles/360020559371-How-do-I-make-a-test-donation-on-my-donation-form-

**Comment by jasloe** [comment]
Huh, I guess this was already circulated as an idea and I'd forgotten about it, #351. My take is that what you described in that ticket is the bulletproof approach. Moreover, better to spend our limited time creating a nicely designed product than messing around with server configuration.

**Comment by cchristopherhowardd** [comment]
I forgot about #351, too, until a couple of weeks ago. The simpler, the better. Is that bulletproof?

I am out of the office tomorrow and Friday, so I'm not able to moderate design discussion between you and Wendy. I sent her the link to the staging page but haven't received a response. I'm awaiting her approval on the email blast invitation, which I hope to send today.

**Comment by cchristopherhowardd** [comment]
The link to the Artsy auction, for the Auction button, is https://www.artsy.net/americanacademyinrome.

Wendy said that we need a phrase near the bottom of the page that says: Auction powered by [Artsy logo with above link]

![artsy-logo-small](https://user-images.githubusercontent.com/38765846/95369432-6ca04180-08a5-11eb-9d01-db1166dd5e60.jpg)
![artsy-logo-large](https://user-images.githubusercontent.com/38765846/95369434-6ca04180-08a5-11eb-9977-82252e776ad6.jpg)

**Comment by cchristopherhowardd** [comment]
Is it possible to move the three buttons [Donate, Auction, Benefit Program] below the video stream? It would be nice to have the Play arrow for the video stream "above the fold." 

I can't believe I just used the phrase "above the fold."

**Comment by jasloe** [comment]
A static HTML page (or series of pages) is bulletproof. No server to crash! I really do think it's the sane approach. Shall I just move forward with it? I could cobble together a working model in a day and refine the design as we go.

> I forgot about #351, too, until a couple of weeks ago. The simpler, the better. Is that bulletproof?
> 
> I am out of the office tomorrow and Friday, so I'm not able to moderate design discussion between you and Wendy. I sent her the link to the staging page but haven't received a response. I'm awaiting her approval on the email blast invitation, which I hope to send today.

**Comment by jasloe** [comment]
> Is it possible to move the three buttons [Donate, Auction, Benefit Program] below the video stream? It would be nice to have the Play arrow for the video stream "above the fold."

Sure.

> I can't believe I just used the phrase "above the fold."

I know, it's a bummer. But it's an occupational hazard.

**Comment by cchristopherhowardd** [comment]
AAR's Stripe account code is 353905. I don't yet know where this information is needed, but here it is.

**Comment by cchristopherhowardd** [comment]
I see that the Donorbox form at https://staging.aarome.org/benefit doesn't match the look of the form on the Donorbox account page, which has a color and specially written text. The first image is the staging site. The second image is Donorbox account page.

<img width="1192" alt="Staging site form" src="https://user-images.githubusercontent.com/38765846/95375195-49799000-08ad-11eb-93d8-1f64ebdece53.png">

<img width="975" alt="Donorbox account page" src="https://user-images.githubusercontent.com/38765846/95375244-5c8c6000-08ad-11eb-9916-88241f704f9d.png">

The embed codes come from this page: https://donorbox.org/org_admin/forms/199099?embed=embed_form.

Does this instruction make sense? This kind of third-party embed/donation stuff is all new to me.

**Comment by jasloe** [comment]
There was a mixup with the Donorbox sign up. I already had an account at j@minorscience.com, which crossed-up the invite I received from the Academy org account. I'm untangling it now.

**Comment by jasloe** [comment]
Ok, the button is calling the correct form now. 

https://www.aarome.org/benefit-test

**Comment by jasloe** [comment]
> AAR's Stripe account code is 353905. I don't yet know where this information is needed, but here it is.

Looks like it's already set up.

**Comment by cchristopherhowardd** [comment]
Is https://www.aarome.org/benefit-test the page we should be reviewing? Should we abandon https://staging.aarome.org/benefit?

**Comment by jasloe** [comment]
Yes, use https://www.aarome.org/benefit-test. I just cannibalized staging to test auth0 sign-in for library users.

If a more fleshed out demo, wireframe, or whatever would be helpful to make the case for a branded site–either with Drupal or as a static page-let me know and I can knock something out while you are away.

**Comment by cchristopherhowardd** [comment]
Can https://www.aarome.org/benefit-test be made public so that Wendy can see it? The page isn't visible if you're not logged into Drupal.

**Comment by jasloe** [comment]
I just published it.

**Comment by jasloe** [comment]
@cchristopherhowardd –– Any preference whether to contain this project on the main site or go with a standalone static site?

**Comment by cchristopherhowardd** [comment]
The URL for the project is https://www.aarome.org/benefit, which I understand as living on the main site.

**Comment by jasloe** [comment]
Right but the question is whether to run that URL on the current site (entailing a bit of work to ensure the page can support 1,000 simultaneous users) or, instead, build a single page, branded "microsite" as discussed in August (#351) and, briefly, last week: https://github.com/aarome/aarome/issues/359#issuecomment-705093269

> Also, restating that the live stream embed as provided by Point seems to work fine. We still need to test the entire Donorbox payment process with a live or dummy payment gateway.
> 
> Considering an alternative so not to burn a bunch of time messing with server configuration:
> 
> * Agree on new design direction (in the next day or two) using Open flyer as basis for a single page "brochure" site for the benefit, including live stream, donation popup, program information
> * Hand code above and post as static HTML file served from an S3 bucket at https://www.aarome.org/benefit
> * Advantage of above is there are no moving parts: just vanilla html/css/javascript. No database connections, no memory or CPU issues.

**Comment by cchristopherhowardd** [comment]
Would the microsite be a subdomain (e.g., benefit.aarome.org)? That would be fine, as long as we can reroute all traffic from aarome.org/benefit to the microsite when the time comes.

**Comment by cchristopherhowardd** [comment]
Or would the URL be something like what we have for the email blasts? For example, https://s3-eu-west-1.amazonaws.com/cdn.aarome.org/html-emails/html/101220-virtual-benefit.html

(That's a Frankenstein email blast that I created this afternoon.)

**Comment by jasloe** [comment]
Frankenstein!

In short, we can serve anything from anywhere. The microsite needn't live outside of the aarome.org domain. The URL would be aarome.org/benefit though it would not pass through the Drupal infrastructure. 

To be clear, the benefits of the microsite idea are:

- less time fussing with live Drupal site
- less time spent on server configuration
- static HTML, no database or web server to break
- frees up time to focus on what matters, in this case a well-designed presentation

**Comment by jasloe** [comment]
Another question: do we need a translated page at /benefit in either case?

**Comment by cchristopherhowardd** [comment]
Let's do the microsite then, at aarome.org/benefit. There is no need for a translated page. There is only one benefit. There is only one page.

**Comment by jasloe** [comment]
Disadvantage is that this process would entail collectively making design decisions which can sometimes (always?) slow things down. Manageable but worth mentioning.

**Comment by cchristopherhowardd** [comment]
I don't see how collectively made design decisions is unavoidable no matter where the page lives. In fact, what the page looks like and if/how it will work on October 27 is really the only thing development cares about.

**Comment by jasloe** [comment]
@cchristopherhowardd –– can you see "Greetings." at https://staging.aarome.org/benefit?

This is where I'll be posting the static site.

**Comment by jasloe** [comment]
@cchristopherhowardd –– Trying to figure out the typeface for One Moment – Infinite Ideas. Minion doesn't seem to match. 

https://fonts.adobe.com/fonts/minion

Can we get this info from Open?

**Comment by jasloe** [comment]
Cool, thank you! 

Posted some stuff. I'll hack out stanzas for the program, auction, credits, etc. tomorrow. 

I'm snapping the video to the top of the viewport so that users can scroll while still seeing the video. Could be minimized further and placed to left or right....welcome your thoughts. 

Could also use some help hashing out a menu system. Let me know when you have some free time to chat tomorrow.

**Comment by cchristopherhowardd** [comment]
I am available for a chat at any time today.

**Comment by cchristopherhowardd** [comment]
The Virtual Benefit streaming page is the absolute most important thing to work on this week. Is it possible to present a draft page at https://staging.aarome.org/benefit/ within the next day or two?

**Comment by jasloe** [comment]
It's already in progress there. You can't see the page?

**Comment by cchristopherhowardd** [comment]
I only see the header text and big box for the video at https://staging.aarome.org/benefit/. I don't see the three buttons or the dummy text that used to appear under the video in previous versions.

**Comment by cchristopherhowardd** [comment]
Here's the top of the page.

<img width="1371" alt="Screen Shot 2020-10-19 at 2 41 35 PM" src="https://user-images.githubusercontent.com/38765846/96497838-42417300-1219-11eb-8039-3a42a218845c.png">

When I scroll down a little, I see this.

<img width="1373" alt="Screen Shot 2020-10-19 at 2 41 46 PM" src="https://user-images.githubusercontent.com/38765846/96497872-4ff6f880-1219-11eb-812b-08eb70a927b0.png">

**Comment by jasloe** [comment]
Make the browser narrower and you'll see the rest of the page. 

I'll tweak the width of the video iframe to make it usable at full width. 

Working on the menu system today as discussed on Thursday.

**Comment by cchristopherhowardd** [comment]
I see the rest of the page, but it's really awkward, as you'll see below.

<img width="1065" alt="Screen Shot 2020-10-19 at 3 11 09 PM" src="https://user-images.githubusercontent.com/38765846/96500978-b0883480-121d-11eb-8a3d-b7814709da72.png">

<img width="746" alt="Screen Shot 2020-10-19 at 3 11 41 PM" src="https://user-images.githubusercontent.com/38765846/96500991-b41bbb80-121d-11eb-9032-7e17ba86f94c.png">

**Comment by cchristopherhowardd** [comment]
The two portrait photos should probably be thumbnails. The VB streaming page ought to have minimal content apart from the header, video portal, and the three buttons.

**Comment by jasloe** [comment]
More WIP on /benefit. Working with Point at the moment to figure out options for player. 

Various spacing tweaks ahead as well as an up arrow at "Up" and a better mobile menu experience.

**Comment by jasloe** [comment]
@cchristopherhowardd –– do you happen to have a contact for Donorbox? Looking for a solution to launch PayPal payment process in a new window with the target attribute.

**Comment by cchristopherhowardd** [comment]
Jesse Sanchez Jr.
jesse@donorbox.org

**Comment by jasloe** [comment]
Pushed spacing cleanup, dropped in real Vimeo link provided by Point. May need to clear browser cache.

**Comment by jasloe** [comment]
No dice on PayPal spawning in new window. Looking at the API and markup I can see why now....the Donorbox form handler collects the donor's information, then securely passes that information to the user's PayPal page. There's no way to pass that information to a new window.

**Comment by cchristopherhowardd** [comment]
We may have to live with PayPal taking people offsite, with the hope they can find their way back to video stream.

**Comment by jasloe** [comment]
I looked more deeply at the PP docs and there is capacity in their API...asked DB for clarification.

**Comment by cchristopherhowardd** [comment]
Great news!

Is there a way for me to made edits to the text in the Honorees and Auction sections?

**Comment by jasloe** [comment]
Not really. It's just static HTML. Think a Google Doc is the best bet....that way you can keep it up to date.

**Comment by cchristopherhowardd** [comment]
A Google Doc is OK with me.

**Comment by cchristopherhowardd** [comment]
Should the Google Doc be in plain text, so that I can type in HTML coding for <em> and <strong> and the like?

**Comment by jasloe** [comment]
Just make it rich text. I can convert it easily.

**Comment by cchristopherhowardd** [comment]
Below is a draft PDF of the Virtual Benefit program, which should be linked to the Program button near top right of the page. The PDF should open in a new tab/window, so that the video streaming isn't interrupted.

I expect the final_final_final_final PDF to be finalized on Monday or even early Tuesday, so we'll need to switch out the draft quickly when the time comes.

[ad73_galaprogram_102120ck_rev.pdf](https://github.com/aarome/aarome/files/5418703/ad73_galaprogram_102120ck_rev.pdf)

**Comment by jasloe** [comment]
Added to staging. May need to clear browser cache. Renamed file to one-moment-infinite-ideas-benefit-program.pdf

**Comment by cchristopherhowardd** [comment]
OK. Can you change the start time on the video portal to 7:30? Right now it says 8:00 and that's incorrect. Thanks!

**Comment by cchristopherhowardd** [comment]
My AAR colleague Wendy moved to Colorado last month. She says her start time is 6:00 PM, so it seems like local time zone plays a factor on the time that's posted here. Is there a way to make the text read 7:30 PM Eastern Time? And also program a variable so that the correct time in each time zone is listed. If not, perhaps the variable time zone function can be disabled so that the video portal just says 7:30 PM Eastern Time?

**Comment by cchristopherhowardd** [comment]
Artsy logos for the Auction section are below:

![artsy-logo-white-png-transparent](https://user-images.githubusercontent.com/38765846/96907416-30560f00-1469-11eb-9680-700d79054895.png)

![artsy-logo-large](https://user-images.githubusercontent.com/38765846/96907501-4bc11a00-1469-11eb-97ae-619a5ab371cc.jpg)
![artsy-logo-small](https://user-images.githubusercontent.com/38765846/96907502-4bc11a00-1469-11eb-8d89-715a7e528554.jpg)

**Comment by jasloe** [comment]
Yes to all!

**Comment by jasloe** [comment]
Now that I see the program and understand what's happening with that, maybe we should remove the host committee section but keep the honorees? Shall I use the Beard/Gates text in the program for the honorees section?

**Comment by cchristopherhowardd** [comment]
Wendy reviewed the body text and did not comment on the presence of the host committee section. I'll ask her specifically about it, but she didn't object to it yesterday.

**Comment by jasloe** [comment]
Here is the performance testing graph for /benefit. 500 concurrent users ramping up over the first 5 minutes. Zero errors, good page load time (average under 1 second).

![benefit-flood](https://user-images.githubusercontent.com/1063667/96910005-d0616780-146c-11eb-8c1e-8125aed3bb4c.png)

**Comment by cchristopherhowardd** [comment]
Graph looks good.

I have the revised body text in a Google Doc at https://docs.google.com/document/d/12VcNqDLHp3EvlAIiLNpL0rYCX89Qvsv7XA10CWobpSs/edit?usp=sharing

**Comment by jasloe** [comment]
> Graph looks good.
> 
> I have the revised body text in a Google Doc at https://docs.google.com/document/d/12VcNqDLHp3EvlAIiLNpL0rYCX89Qvsv7XA10CWobpSs/edit?usp=sharing

Why not just use what's in the program for consistency?

**Comment by cchristopherhowardd** [comment]
I can't vouch for the contents of the program.

**Comment by jasloe** [comment]
Point controls the iframe content.

> My AAR colleague Wendy moved to Colorado last month. She says her start time is 6:00 PM, so it seems like local time zone plays a factor on the time that's posted here. Is there a way to make the text read 7:30 PM Eastern Time? And also program a variable so that the correct time in each time zone is listed. If not, perhaps the variable time zone function can be disabled so that the video portal just says 7:30 PM Eastern Time?

**Comment by cchristopherhowardd** [comment]
One addition to the Host Committee section, to be placed at the very end: Fred Wilson (2020 Resident) and Whitfield Lovell (2019 Resident)

**Comment by jasloe** [comment]
Pushed a few things to staging. 

Sent you a few notes via email.

**Comment by cchristopherhowardd** [comment]
I don't see the revised text from the Google Doc at https://staging.aarome.org/benefit/

**Comment by cchristopherhowardd** [comment]
For example, the list of Host Committee members doesn't match what's in the Google Doc. The list of auction people doesn't match. (There's all that RAAR and FAAR garbage, too.) The bios for Mary Beard and Theaster Gates changed and got longer.

**Comment by jasloe** [comment]
Made a judgement call and used the program text which seems stet. Guess it is not. The text for Beard, for example, was just something I pulled from various sources. As such, first sentence of each paragraph for Beard are weird. Why mention her professorship twice?

**Comment by jasloe** [comment]
Didn't notice second pages of GDoc. Will fix now.

**Comment by cchristopherhowardd** [comment]
I revised the second paragraph of Beard, so that's fixed now. This webpage isn't supposed to replicated the program; it's intended to have minimal content--a couple of bells but no whistles--and serve simply as a place to stream the video and have buttons that link to a few other things.

**Comment by jasloe** [comment]
Cool. I fixed the copy. Need to add a few classes for the links in the auction section and two spacing issues. DId you see my correspondence with Jesse re: Paypal? I'm increasingly annoyed by the issue and think it's going to cause problems to bounce people out of the live stream.

**Comment by cchristopherhowardd** [comment]
The Beard and Gates texts are basically pilfered from other online sources. It's an ongoing problem at the Academy. It's likely that nobody will notice, but I may rewrite and minimize their bios.

**Comment by cchristopherhowardd** [comment]
The Artsy logo (at the end of the sentence "Auction powered by") should have a link to https://artsy.net/americanacademyinrome

**Comment by cchristopherhowardd** [comment]
Can you add these three lines to the Host Committee list? They should appear alphabetically with aplomb--under Polsky, Tsien, and Weisberg.

Cyntha Hazen Polsky and Leon Polsky
Billie Tsien (2000 Resident) and Tod C. Williams (1983 Fellow)
Leah and Michael Weisberg

**Comment by jasloe** [comment]
You bet. What time should I move this to production? Sometime this evening?

________________________________

**Comment by cchristopherhowardd** [comment]
Let's do it after 6:00 PM.

**Comment by jasloe** [comment]
Added. Since I have to leave early today and just to get this started, I am going to move this over to aarome.org now and put it at /benefit-test. When we're ready to go live, it's just a matter of changing the directory name. Will let you know when it's done.

**Comment by jasloe** [comment]
Ok, we're over at aarome.org/benefit-test now. I am waiting for another iframe from Point and will run one more traffic test. Will also fix the Artsy logo link.

**Comment by cchristopherhowardd** [comment]
Two tiny edits on first paragraph for Mary Beard: the titles of the two publications in this sentence (Times Literary Supplement, A Don's Life) should be italicized.

Since 1992 she has been the classics editor of the _Times Literary Supplement_, where she also writes a regular blog, _A Don’s Life_.

**Comment by cchristopherhowardd** [comment]
When you move aarome.org/benefit-test to aarome.org/benefit tonight, remember to delete the link forwarder that currently sends people to https://www.aarome.org/support/virtual-benefit!

**Comment by cchristopherhowardd** [comment]
Here is the final PDF of the gala program, to replace the current PDF on the page.

[OneMomentInfiniteIdeas_Program2020.pdf](https://github.com/aarome/aarome/files/5446334/OneMomentInfiniteIdeas_Program2020.pdf)

**Comment by cchristopherhowardd** [comment]
Can you let me know ASAP when you upload the new PDF?

**Comment by jasloe** [comment]
Posted.

> Can you let me know ASAP when you upload the new PDF?

**Comment by jasloe** [comment]
Restored virtual benefit static page at /benefit.
